lib/ansible/modules/network/aci/aci_anp.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+
+module: aci_anp
+short_description: Manage top level application network profile objects
+description:
+    -  Manage top level application network profile object, i.e. this does
+      not manage EPGs.
+author:
+- Swetha Chunduri (@schunduri)
+- Dag Wieers (@dagwieesrs)
+version_added: '2.4'
+requirements:
+    - ACI Fabric 1.0(3f)+
+notes: Tenant must exist prior to using this module
+options:
+   tenant_name:
+     description:
+     - The name of the tenant 
+     required: yes
+   app_profile_name:
+     description:
+     - The name of the application network profile
+     required: yes
+   description:
+     description:
+     - Description for the ANP
+   state:
+     description:
+     - present, absent, query
+     default: present
+     choices: [ absent, present, query ]
+extends_documentation_fragment: aci
+'''
+
+EXAMPLES = r'''
+
+- name: Add a new ANP
```

Comment:
For the examples I think it's better to add proper input as if this is a real-world action.
If you put text (with spaces) in there, it will confuse people as to what input is allowed.
And just adding the description text for the parameter does not give any valuable information.